tests/test_continue_training.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+
+# (C) Copyright 2020, 2021, 2022, 2023, 2024 IBM. All Rights Reserved.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests for continuing to train."""
+
+from torch import manual_seed, rand, mean, tensor, allclose
+from torch.nn import Linear
+
+from aihwkit.optim.analog_optimizer import AnalogSGD
+from aihwkit.simulator.configs import TorchInferenceRPUConfig
+from aihwkit.simulator.configs.utils import BoundManagementType, NoiseManagementType
+from aihwkit.nn.conversion import convert_to_analog
+
+
```

Comment:
Could you use the tile paramteric tests in tests/helper so that one can easily use this test for all AnalogTile and not just TorchInferenceTile ?  Also there is actually a test that should have triggered this, why was that test passing? See https://github.com/IBM/aihwkit/blob/master/tests/test_utils.py#L200